packages/dcos-history/extra/history/server.py

```diff
@@ -1,93 +1,8 @@
-import logging
 import os
-import sys
+import history.server_util
```

Comment:
> "Not having if __name__ == '__main__' doesn't work for unit tests"
I don't follow. Unit tests weren't invoked by calling the server file as main

What I find awkward here is the whole purpose of this module is to create a thread bound to a global variable *on import*, just so we can invoke gunicorn and have this still work. Possible alternatives:
(1) Use a gunicorn setting like post_worker_init to start the thread within the GIL of the worker and enforce that we only have 1 worker. This is exactly what the history service is doing now (flask only supports a single worker)
(2) Don't use gunicorn at all. It seems to be adding a lot more magic. If the goal of this PR is to catch exceptions and fail hard, then its not clear how this PR accomplishes that other than trusting the gunicorn will fulfill this purpose